### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adamrbehrman**
### First time contributor checklist

- [ ] I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [ ] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [ ] I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [ ] I have tested my contribution on these devices:
 * Device A, Android X.Y.Z
 * Device B, Android Z.Y
 * Virtual device W, Android Y.Y.Z
- [ ] My contribution is fully baked and ready to be merged as is
- [ ] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### CODE REVIEW

**Comment by alsgal2** [comment]
Ok

**Comment by alsgal2** [comment]
I'll give it a try.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Signal Android 
 
-Signal is a messaging app for simple private communication with friends.
+Signal is a messaging app for simple, private communication with friends.
```
Comment (jlund-signal) [inline README.md]: I think that "simple and private" would sound even better.

File: README.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ https://www.transifex.com/projects/p/signal-android/
 ## Contributing Code
 Instructions on how to setup your development environment and build Signal can be found in  [BUILDING.md](https://github.com/signalapp/Signal-Android/blob/master/BUILDING.md).
 
-If you're new to the Signal codebase, we recommend going through our issues and picking out a simple bug to fix (check the "easy" label in our issues) in order to get yourself familiar. Also please have a look at the [CONTRIBUTING.md](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md), that might answer some of your questions.
+If you're new to the Signal codebase, we recommend going through our issues and picking out a simple bug to fix (check the "easy" label in our issues) in order to get yourself familiar. Also please have a look at the [CONTRIBUTING.md](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md). That might answer some of your questions.
```
Comment (jlund-signal) [inline README.md]: We need to either add a word after "CONTRIBUTING.md" or remove the "the" that precedes it. That sentence doesn't really work right now.

Alternatives:
* Also, please have a look at the [CONTRIBUTING.md](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) document.
* Also, please have a look at [CONTRIBUTING.md](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md).

I think I prefer the first.

**Comment by alan-signal** [comment]
Closing. Reason: Requested changes were not applied.

We provided feedback or changes were requested, and that was not acted upon in a reasonable time.

**Pull request closed without merging**